php/php.editor/test/unit/data/testfiles/completion/lib/_base/issue153707.php.testIssue153707_01.completion

```diff
@@ -4,7 +4,11 @@ echo "class property: |$this->name\n";
 PACKAGE    ANS                             [PUBLIC]   null
 PACKAGE    CNS                             [PUBLIC]   null
 CLASS      Book                            [PUBLIC]   issue153707.php
+METHOD     getName()                       [PUBLIC]   Book
+METHOD     setName($name)                  [PUBLIC]   Book
 VARIABLE   ? $name                         [PUBLIC]   issue153707.php
+VARIABLE   ? name                          [PUBLIC]   Book
```

Comment:
Is this expected?

Reply:
Yes, this `name` variable means `$this->name`. `$name` is the parameter. 
https://github.com/apache/netbeans/blob/fb38c4ec8b578efd70fc84d81fcff8a16b495deb/php/php.editor/test/unit/data/testfiles/completion/lib/_base/issue153707.php#L3-L19

(But we should improve this case because there are unnecessary CC items.)

Reply:
OK, thanks.